website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -1,37 +1,37 @@
+Once the Microsoft Entra ID app is added to dbt Cloud, an account admin must also connect a service user via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
 1. [Connecting a Service User](#connecting-a-service-user).
 
 
-Once the Azure AD app is added to dbt Cloud and the service user is connected, then dbt Cloud developers can personally authenticate in dbt Cloud from Azure DevOps. For more on this, see [Authenticate with Azure DevOps](/docs/cloud/git/authenticate-azure).
+Once the Microsoft Entra ID app is added to dbt Cloud and the service user is connected, then dbt Cloud developers can personally authenticate in dbt Cloud from Azure DevOps. For more on this, see [Authenticate with Azure DevOps](/docs/cloud/git/authenticate-azure).
 
-## Register an Azure AD app
+## Register a Microsoft Entra ID app
 
-1. Sign into your Azure portal and click **Azure Active Directory** under Azure services.
+1. Sign into your Azure portal and click **Microsoft Entra ID** under Azure services.
 2. Select **App registrations** in the left panel.
-3. Select **New registration**. The form for creating a new Active Directory app opens.
+3. Select **New registration**. The form for creating a new Microsoft Entra ID app opens.
 4. Provide a name for your app. We recommend using, "dbt Labs Azure DevOps App".
-5. Select **Accounts in any organizational directory (Any Azure AD directory - Multitenant)** as the Supported Account Types.
-Many customers ask why they need to select Multitenant instead of Single tenant, and they frequently get this step wrong. Microsoft considers Azure DevOps (formerly called Visual Studio) and Azure Active Directory as separate tenants, and in order for this Active Directory application to work properly, you must select Multitenant.
-6. Add a redirect URI by selecting **Web** and, in the field, entering `https://YOUR_ACCESS_URL/complete/azure_active_directory`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
+5. Select **Accounts in any organizational directory (Any Microsoft Entra ID directory - Multitenant)** as the Supported Account Types.
+Many customers ask why they need to select Multitenant instead of Single tenant, and they frequently get this step wrong. Microsoft considers Azure DevOps (formerly called Visual Studio) and Microsoft Entra ID as separate tenants, and in order for this Microsoft Entra ID application to work properly, you must select Multitenant.
+6. Add a redirect URI by selecting **Web** and, in the field, entering `https://YOUR_ACCESS_URL/complete/microsoft_entra_id`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
 7. Click **Register**.
 
 <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/ADnavigation.gif" title="Navigating to the Azure AD app registrations"/>
```

Comment:
should it be ms entra id?
```suggestion
<Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/ADnavigation.gif" title="Navigating to the Microsoft Entra ID app registrations"/>
```

Reply:
logging for posterity: chatted with docs team about the two options (mismatched caption/image vs still referencing AAD). as a group, we leaned towards first option